cmd/cost/list.go

```diff
@@ -61,7 +86,7 @@ func newListOptions(streams genericclioptions.IOStreams) *listOptions {
 }
 
 //List the cost of each OU under given OU
-func listCostsUnderOU(OU *organizations.OrganizationalUnit, awsClient awsprovider.Client, ops *listOptions) error {
+func (o *getOptions) listCostsUnderOU(OU *organizations.OrganizationalUnit, awsClient awsprovider.Client, ops *listOptions) error {
```

Comment:
Did you mean to use ListOptions here?

Reply:
the func listCostsUnderOU calls a func from get.go. I was wanting it to inherit the struct from getOptions. Refactoring this to make it more readable